Makefile

```diff
@@ -58,12 +58,28 @@ lint:
 man:
 	man/md2man-all.sh
 
-runcimage:
+runcimage: | .busybox .hello-world .debian
 	$(CONTAINER_ENGINE) build $(CONTAINER_ENGINE_BUILD_FLAGS) -t $(RUNC_IMAGE) .
 
+.hello-world:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create hello-world) -o $(CURDIR)/tests/integration/testdata/hello-world.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/hello-world.tar
+	touch "$@"
+
+.busybox:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create busybox) -o $(CURDIR)/tests/integration/testdata/busybox.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/busybox.tar
+	touch "$@"
+
+.debian:
+	$(CONTAINER_ENGINE) export $(shell $(CONTAINER_ENGINE) create debian) -o $(CURDIR)/tests/integration/testdata/debian.tar
+	chmod a+r $(CURDIR)/tests/integration/testdata/debian.tar
+	touch "$@"
```

Comment:
1. You need to add something like `rm -f .debian .busybox .hello-world` to `clean` target. The downloaded images, too, I guess.

2. Also, it might be helpful to do
```
ln -sf $(CURDIR)/tests/integration/testdata/debian.tar $@
```
instead of just `touch`, as in the case a tarball will be removed, `make` will redo the target thanks to a broken symlink.

3. Using $(CURDIR) is redundant I think.

4. These three rules look similar, and it should be easy to generalize and merge those into one single rule.

5. Maybe move this code out to a separate Makefile in tests/integration/testdata, and just call it from the main Makefile.